src/router.js

```diff
@@ -17,8 +17,14 @@ export default function (options) {
       }
     },
     hooks: {
+      onUpdate: function (oldModel, newModel, data, actions) {
+        if(newModel.router && oldModel.router.path !== newModel.router.path &&
```

Comment:
This seems to rely on `newModel.router` being a different object than `oldModel.router`. As far as I know we don't guarantee that.